src/components/ActionSheetItem/ActionSheetItem.tsx

```diff
@@ -40,6 +40,10 @@ export interface ActionSheetItemProps
    */
   onClick?: React.MouseEventHandler<HTMLElement>;
   onImmediateClick?: React.MouseEventHandler<HTMLElement>;
+  /**
+   * Иконка для `checked` режима.
+   */
+  CheckedIcon?: React.ComponentType<Pick<React.AriaAttributes, "aria-hidden">>;
```

Comment:
```suggestion
  IconСhecked?: React.ComponentType<Pick<React.AriaAttributes, "aria-hidden">>;
```

Вообще можно было бы сделать `React.ComponentType<typeof Icon20CheckCircleOn>` (к сожалению, `@vkontakte/icons` пока не эспортирует `SvgIconProps`), чтобы потом не делать брейкинг чендж если понадобиться ещё и `className` или другие SVG пропсы передавать

Reply:
В стилях, кстати 24x24 зашито 🤔 

<img width="269" alt="image" src="https://user-images.githubusercontent.com/5850354/194549343-63c25326-d532-4f3a-9a2f-d404260f108f.png">

Reply:
Вообщем, я решил сделать React.ReactNode. Так должно доставлять меньше проблем

Стили пофиксил